src/components/Auth/Auth.tsx

```diff
@@ -0,0 +1,92 @@
+import { h, FunctionComponent } from 'preact'
+import { AuthCheckProcessor } from './AuthCheckProcessor'
+import { FaceTecSDK } from '~auth-sdk/FaceTecSDK.js/FaceTecSDK'
+import { Config } from './AuthConfig'
+import { FaceTecStrings } from './assets/FaceTecStrings'
+import type { WithLocalisedProps } from '~types/hocs'
+import type { StepComponentBaseProps } from '~types/routers'
+import { useEffect, useState } from 'preact/hooks'
+
+type Props = StepComponentBaseProps & WithLocalisedProps
+
+const AuthCapture: FunctionComponent<Props> = ({
+  token,
+  nextStep,
+  translate,
+  events,
+  customUI,
+}) => {
+  const [authConfig, setAuthConfig] = useState({
+    token: '',
+    production_key_text: '',
+    device_key_identifier: '',
+    public_key: '',
+  })
```

Comment:
While the type of `authConfig` will be inferred with `useState()` default value, I still suggest explicitly declare a `type` of that to:
- Tell what are the possible values of the state
- Give type checking in case we need to add new properties